cardano-db-sync/src/Cardano/DbSync/Era.hs

```diff
@@ -101,7 +101,8 @@ mkShelleyTopLevelConfig sgc =
     -- Can use Nothing for the last field because that field will be dropped
     -- by 'pInfoConfig' anyway.
     shelleyInfo :: ProtocolInfo IO ShelleyBlock
-    shelleyInfo = protocolInfoShelley sgc (ProtVer 0 0) Nothing
+    -- TODO: IS @'maxMajorPV' = 0@ THIS RIGHT?
+    shelleyInfo = protocolInfoShelley sgc 0 (ProtVer 0 0) Nothing
```

Comment:
@erikd I don't know what value should be passed here, probably something else than `0`.

Reply:
Neither do I. Complete mystery to me.

Reply:
@mrBliss @nc6 any idea what should be put here?

Reply:
```suggestion
    shelleyInfo = protocolInfoShelley sgc 2 (ProtVer 2 0) Nothing
```
I believe that the major protocol version for Shelley will be 2, there was a discussion about it on Slack (OBFT is 1.0.0).

The `maxMajorPV` is the maximum major protocol version that is allowed by the ledger rules. I don't know what we should use, but it should at least be >= the major version of Shelley. Maybe @JaredCorduan knows a good value for `maxMajorPV`?